tensorflow_serving/model_servers/server.cc

```diff
@@ -284,6 +292,10 @@ Status Server::BuildAndStart(const Options& server_options) {
     return errors::InvalidArgument("Failed to BuildAndStart gRPC server");
   }
   LOG(INFO) << "Running gRPC ModelServer at " << server_address << " ...";
+  if (!grpc_socket_path.empty()) {
+    LOG(INFO) << "Running gRPC ModelServer at UNIX socket "
```

Comment:
can you please add a test (attempt calling one of the api endpoints, say "predict") to:

serving/tensorflow_serving/model_servers/tensorflow_model_server_test.py

the (test) server listens on tcp+http ports, have it also listen on UDS. and then add
a new test method (say testPredictUDS) to test on UDS. no need to add tests for
each of the API end points. this is mostly to check if grpc/uds is functional.

thanks!